draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -207,7 +207,7 @@ Note: In some credential types, it is not desirable to include an expiration dat
 
 ## Issuer identification and key resolution to validate an issued Credential {#issuer-key-resolution}
 
-This profile supports two ways to represent and resolves the key required to validate the issuer signature of a SD-JWT VC, web PKI-based key resolution and x.509 certificates.
+This profile supports two ways to represent and resolves the key required to validate the issuer signature of an SD-JWT VC, web PKI-based key resolution and x.509 certificates.
 
 * Web-based key resolution: The key used to validate the Issuer’s signature on the SD-JWT VC MUST be obtained from the SD-JWT VC issuer's metadata as defined in Section 5 of [@!I-D.terbu-sd-jwt-vc]. The JOSE header `kid` MUST be used to identify the respective key.
```

Comment:
unfortunately I-D.terbu-sd-jwt-vc doesn't support OIDC Federation, that's a webpki with trust chains and a solid alternative to X.509

Reply:
nothing in the sd-jwt vc draft precludes use of OIDC Federation, in the same way as it does not preclude x.509
HAIP mandates jwt issuer and x.509, adding another (mandatory to implement) option seems a complexity driver, but let's talk. Please open an issue about it as this PR is about a different topic.

Reply:
Done here https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/issues/59

Reply:
thanks